patches/react-native+0.69.11.patch

```diff
@@ -1,50 +0,0 @@
-diff --git a/node_modules/react-native/React/CoreModules/RCTTiming.mm b/node_modules/react-native/React/CoreModules/RCTTiming.mm
-index 13d0d57..00e5d4c 100644
---- a/node_modules/react-native/React/CoreModules/RCTTiming.mm
-+++ b/node_modules/react-native/React/CoreModules/RCTTiming.mm
-@@ -127,7 +127,15 @@ - (void)setup
- {
-   _paused = YES;
-   _timers = [NSMutableDictionary new];
--  _inBackground = NO;
-+    
-+  __block BOOL initialInBackground;
```

Comment:
I don't think RN does this out of the box. We need this patch to be applied to the new version.